db/schema.rb

```diff
@@ -548,9 +599,10 @@
     t.integer "role", default: 0
     t.text "authentication_token"
     t.datetime "authentication_token_created_at"
+    t.integer "municipality_id"
     t.index ["authentication_token"], name: "index_users_on_authentication_token", unique: true, length: 255
     t.index ["confirmation_token"], name: "index_users_on_confirmation_token", unique: true
-    t.index ["email"], name: "index_users_on_email", unique: true
+    t.index ["email", "municipality_id"], name: "index_email_on_municipality_id"
```

Comment:
please remove the schema file entirely. it is not need and provides informations, that are not relevant for the master here.